src/main/java/net/glowstone/block/itemtype/ItemType.java

```diff
@@ -130,15 +130,17 @@ protected final void setMaxStackSize(int maxStackSize) {
      * @param holding The ItemStack the player was holding
      */
     public void rightClickAir(GlowPlayer player, ItemStack holding) {
-        // nothing by default
+        if (fastEquip) {
+            equipStack(player, holding);
+        }
     }
 
     /**
      * If this item can only be used without any context (essentially used in air)
      * @return If this item can only be used without any context
      */
     public boolean canOnlyUseSelf() {
-        return false;
+        return fastEquip;
```

Comment:
This is for the ability to equip armor by right clicking it, as I remember checking and it not being implemented.